CHANGELOG.md

```diff
@@ -7,20 +7,27 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 - No changes yet.
 
+## [1.26.0] - 2021-02-18
+### Changed
+- Codegeneration for typedefs now uses use generated `MarshalLog...` functions
+  where appropriate to avoid casting to a root go type from go packages of
+  transitive thrift dependencies.
+- Rewrote internal wire's unsafeStringToBytes to adhere to 1.16 vet check.
+
```

Comment:
Is this supposed to be part of 1.26.0 release?

Reply:
I followed the steps in RELEASE.md, which made these changes land on master but not on dev yet (see https://github.com/thriftrw/thriftrw-go/commit/aeb92434ba42289cef33c641c9c2104ffec1af89 for example).

Reply:
ohh that makes sense then! Thanks for explaining.